bigquery/README.md

```diff
@@ -4,3 +4,37 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+SQL queries through the BigQuery CLI:
+
+bq query '\<SQL QUERY\>'
```

Comment:
Can you put this in a block:

```markdown
```bash
$ kind_of like_this
```
```

so that it gets formatted nicely?

See https://guides.github.com/features/mastering-markdown/